packages/proof-service/src/domain/create-proof.ts

```diff
@@ -33,13 +42,29 @@ export async function createProof(taskId: string) {
   try {
     const circuitName = `${task.database}.${task.collection}`;
     const modelDbSetting = ModelDbSetting.getInstance();
-    const { merkleHeight, appPublicKey } =
+    const { merkleHeight, appPublicKey, networkId } =
       (await modelDbSetting.getSetting(task.database)) || {};
 
     if (!merkleHeight || !appPublicKey) {
       throw new Error('Setting is wrong, unable to deconstruct settings');
     }
 
+    const network = await ModelNetwork.getInstance().findOne({
+      id: networkId,
+      active: true,
+    });
+
+    if (network) {
+      throw Error('No active network found');
+    }
+
+    Mina.setActiveInstance(
+      Mina.Network({
+        networkId: network!.id as NetworkId,
+        mina: network!.endpoint,
+      })
+    );
```

Comment:
network checked already so don't need to be check

Reply:
done